spec/models/data_source_spec.rb

```diff
@@ -7,6 +7,14 @@
       is_expected.to validate_presence_of(:api)
       is_expected.to validate_presence_of(:data)
     end
+
+    ['foo bar', 'de-minimis', "loren's", 'Currencies'].each do |bad_value|
+      it { should_not allow_value(bad_value).for(:api) }
```

Comment:
I suggest you use RSpec's newer `is_expected.to` syntax for one-liners. Even though "should" one-liners are not deprecated in RSpec 3 (as opposed to the should method money patch which is), using `is_expected.to` will make your changes consistent with the rest of the codebase.